docs/meshstack.openshift.index.md

```diff
@@ -217,6 +217,28 @@ oc get serviceaccount meshfed-service -o json | jq '.secrets[].name' | grep toke
 
 Operators need to securely inject this access token into the configuration of the OpenShift module.
 
+### Custom Account Roles
+
+If you want to use custom account roles (and not the pre-defined `admin`, `edit` and `view`) you need to make sure to also list these roles in the clusterrole binding section for the meshfed-service principle. It is not allowed for the service-principle to grant roles granting more rights then itself has, so the right to bind these roles must be explicitly granted.
```

Comment:
you mention "account roles" which is a term we don't use anywhere else yet - can you document this as meshProject roles and also add links to the relevant page explaining meshProject roles?

Reply:
yes also improved the wording a bit.